src/comparable.cr

```diff
@@ -1,22 +1,22 @@
 # The `Comparable` mixin is used by classes whose objects may be ordered.
 #
 # Including types must provide an `<=>` method, which compares the receiver against
-# another object, returning `-1`, `0`, or `+1` depending on whether the receiver is less than,
-# equal to, or greater than the other object.
+# another object, returning a negative number, zero, or a positive number depending
+# on whether the receiver is less than, equal to, or greater than the other object.
 #
 # `Comparable` uses `<=>` to implement the conventional comparison operators (`<`, `<=`, `==`, `>=`, and `>`).
 module Comparable(T)
-  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns `-1`.
+  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns a negative number.
   def <(other : T)
     (self <=> other) < 0
   end
 
-  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns `-1` or `0`.
+  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns a negative number or zero.
   def <=(other : T)
     (self <=> other) <= 0
   end
 
-  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns `0`.
+  # Compares this object to *other* based on the receiver’s `<=>` method, returning `true` if it returns zero.
```

Comment:
I'd prefer to use `0` instead of "zero" everywhere. It's an explicit value and we should use the appropriate literal for that.

Reply:
I used "zero" because "negative number" and "positive number" is also written in words instead of `-1` and `+1`, so for consistency. But "negative number" and "positive number" is very general while `0` is a unique value so they are still pretty different and `0` reads faster, so I will change that.